test/conftest.py

```
@@ -37,3 +43,114 @@
+    3. Constructs the RTSP server address.
+    4. Constructs the ffmpeg command to stream the input file over RTSP using ffmpeg-python.
+    5. Starts the ffmpeg process in the background.
+    6. Waits for a short period to allow the server to start.
+    7. Yields the RTSP server address to the test function.
+    8. After the test function completes, it stops the ffmpeg process.
+
+    Returns:
+        str: The RTSP server address.
+    """
+
+    def is_port_free(port):
```

Ruff (0.11.9)

Missing return type annotation for private function `is_port_free`

Add return type annotation: `bool | None`

(ANN202)